src/Task01_MaxSubarraySum/MaxSubarraySum.js

```diff
@@ -1,34 +1,33 @@
-"use strict";
+'use strict'
 
 // O(n^2) complexity
-function getMaxSubSum(arr) {
-  let maxSum = 0;
-
+function getMaxSubSum (arr) {
+  let maxSum = 0
   for (let i = 0; i < arr.length; i++) {
-    let currentSum = 0;
+    let currentSum = 0
     for (let j = i; j < arr.length; j++) {
-      currentSum += arr[j];
-      maxSum = Math.max(maxSum, currentSum);
+      currentSum += arr[j]
+      maxSum = Math.max(maxSum, currentSum)
     }
   }
 
-  return maxSum;
+  return maxSum
 }
 
 // O(n) Linear complexity
-/*function getMaxSubSum(arr) {
-  let maxSum = 0;
-  let currentSum = 0;
+/* function getMaxSubSum (arr) {
```

Comment:
uncomment, change name, add to export and add tests. that should work the same as the metod above